toml.md

```diff
@@ -161,9 +161,23 @@ In JSON land, that would give you the following structure:
 }
 ```
 
-Whitespace around dot-separated parts is ignored, however, best practice is to
+Whitespace around dot-separated parts is ignored. However, best practice is to
 not use any extraneous whitespace.
 
+```toml
+fruit.name = "banana"     # this is best practice
+fruit. color = "yellow"    # same as fruit.color
+fruit . flavor = "banana"   # same as fruit.flavor
+```
+
+Indentation is treated as whitespace. Tabs and spaces before keys are ignored.
+
+```toml
+bad_fruit.name = "banana"
+  bad_fruit.color = "green"  # Indentation is VALID, BUT DISCOURAGED.
```

Comment:
There is still a DISCOURAGED here, as well as in the CHANGELOG (twice). I certainly agree that the indentation shown in this example would be nonsensical, but indenting all key-value pairs two spaces (say) or indenting whole nested tables depending on the number of name parts they contain seem reasonable stylistic choices.

Reply:
I do agree with you, to an extent. But when the indentation doesn't match the depth of the specified subtable, there could be chaos.

I'll take out both of those commentaries. Not sure what to replace them with, though.

Reply:
Replaced them with nothing. No need for examples.